db/create_db.sql

```diff
@@ -1,8 +1,8 @@
-CREATE DATABASE IF NOT EXIST mew_co;
+CREATE DATABASE IF NOT EXISTS mew_co;
 
-USE DATABASE mew_co;
+USE mew_co;
```

Comment:
why did you remove the declaration?

Reply:
It's a typo I make every single time. The correct command is actually `Use <db_name>`. Using the command there is a syntax error.